ecommerce-iosWidget/WidgetView/ItemsGridView.swift

```diff
@@ -10,14 +10,14 @@ import SwiftUI
 
 struct ItemsGridView: View {
 
-    var items: [String]
+    var items: [ItemViewModel]
 
     let columns = Array(repeating: GridItem(.flexible()), count: 4)
 
     var body: some View {
         LazyVGrid(columns: columns, spacing: 15.0) {
-            ForEach(items, id: \.self) { _ in
-                ItemView(imageName: "star.fill")
+            ForEach(items, id: \.id) { item in
```

Comment:
Because `ItemViewModel` adopted the `Identifiable` protocol, so we can obmit `id: \.id`.

```suggestion
            ForEach(items) { item in
```